keyboards/montsinger/rebound/rebound.h

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2020 Ross Montsinger
+/* Copyright 2023 Ross Montsinger
```

Comment:
```suggestion
/* Copyright 2020 Ross Montsinger
```
This should fix the conflict. This file does not exist anymore.